api/src/main/resources/workbench.yaml

```diff
@@ -201,8 +201,8 @@ paths:
     delete:
```

Comment:
Are we ever going to support deleting your account?

Particularly if we think we will, we might want to change this endpoint to not use DELETE on the User... as you're not really deleting the user, you're just disabling the account.

Reply:
I wasn't planning to, since it messes with auditing, and also creates silent errors if the same username is used for different accounts.

Do you think it should be a post call rather than delete? (I am happy either way I think.)

Reply:
That seems OK to me... @jmandel let me know if you have any different requirements. (I'm used to Google where we guarantee that data deleted by the user eventually gets wiped out entirely... hopefully that's not necessary here.)